README.md

```diff
@@ -43,41 +64,81 @@ Examples:
 * `otio-mxf` (complex plugin to read, write and link MXF files)
 
 
-## Suggested folder structure
-
-```bash
-├── LICENSE
-├── otio_plugin_template
-│   ├── __init__.py
-│   ├── plugin_manifest.json
-│   ├── adapters
-│   │   ├── __init__.py
-│   │   ├── my_adapter.py
-│   ├── hooks
-│   │   ├── __init__.py
-│   │   ├── my_hook.py
-│   ├── operations
-│   │   ├── __init__.py
-│   │   ├── my_media_linker.py
-│   └── schemadefs
-│       ├── __init__.py
-│       ├── my_schemadef.py
-├── README.md
-├── setup.cfg
-├── setup.py
-├── tests
-    └── test_my_plugin.py
+## Default folder structure
+Below is the default file and folder tree that comes with the plugin template.
+  
+```
+ |── LICENSE
+ ├── otio_plugin_template
+ │   ├── __init__.py
+ │   ├── plugin_manifest.json  # Required
+ │   ├── adapters
+ │   │   ├── __init__.py
+ │   │   ├── my_adapter.py
+ │   ├── hooks
+ │   │   ├── __init__.py
+ │   │   ├── my_hook.py
+ │   ├── operations
+ │   │   ├── __init__.py
+ │   │   ├── my_media_linker.py
+ │   └── schemadefs
+ │       ├── __init__.py
+ │       ├── my_schemadef.py
+ ├── README.md
+ ├── setup.cfg
+ ├── setup.py
+ ├── tests
+     └── test_my_plugin.py
+```
+
+### Reorganizing the folder structure to suite your plugin
+You're free to rename, remove or restructure the files and folders to best suite 
+your plugin. Simple adapters may not need a deep folder structure (see example below).  
+Just make sure the `plugin_manifest.json` file is kept and that the contents 
+inside it reflect your choices. This makes sure OpenTimelineIO's plugin system 
+loads your plugin properly.
+
+> **TIP!** Make sure to add a descriptive docstring at the top of your plugin files, so they 
+register properly and inform users of what they do.
+
+Example of a simple adapter plugin. Notice how we removed the "adapters" folder.
+```
+ |── LICENSE
+ ├── otio_my_adapter
```

Comment:
I would still recommend using the `src` layout. Using a `src` folder can avoid some errors while running tests (for example it ensures that tests are not ran against the non installed code (unless you mess with your PYTHONPATH).

Basically, when using the `src` layout, `otio_my_adapter` is not importable when python is invoked from the root of the repo. WHich means that to run the tests in CI, you'll have to actually install the project using `pip install .`, which will test the code and also test that the packaging works as expected (al files are packaged in the wheel, etc).

Reply:
Thanks!
This makes a lot of sense!
I think I'll do the restructuring of code in a separate PR though.